content/events/gcc2024/abstracts/index.md

```diff
@@ -90,13 +68,13 @@ accepted talk will automatically have an option to also present a poster.
 
 Slides for your talk will be projected on a screen in the 16:9 format.
 
-GCC2024 presentations will be streamed live. To support virtual attendees, the
-organizers will need slides for all talks before the event (see [key
-dates](/events/gcc2024/key-dates/)). **We ask that you please create your slides
-in [Google Slides](https://slides.google.com), share the slide deck with *anyone
-with the link* (in view-only mode), and provide a link in [this
-spreadsheet](https://docs.google.com/spreadsheets/d/1-YnNkpP9jQ40SlFno-eemR33-KfjBn-W2vPkC5LhRCk/edit?usp=sharing)**
-by the deadline listed above.
+GCC2024 presentations will be streamed live. To support virtual attendees, **the
+organizers will need slides for all talks by June 15.** We ask that you please
```

Comment:
Is an empty slide deck ok ? If not I think it'd be nice to send out an email early about this.

Reply:
There is no prescribed slide format or anything, just a request to use Google Slides for making the slides so they are easy to share/incorporate.

And yes re the email - I was planning on sending an email later this week (about 3 weeks before the deadline) and then another one about a week ahead of the deadline.